playbook/app/pb_kits/playbook/pb_line_graph/line_graph.rb

```diff
@@ -27,11 +27,16 @@ def chart_type
         gradient ? "area" : "line"
       end
 
+      def dark_theme
+        dark ? "dark" : ""
+      end
+
       def chart_options
         {
           id: id,
           className: classname,
           chartData: chart_data,
+          dark: dark_theme,
```

Comment:
```suggestion
          dark: dark ? "dark" : "",
```

Reply:
Curious why not abstract logic in a method here?

Reply:
Method is 3 lines, not re-used anywhere else. More lines means more vertical eye tracking versus a single ternary. Also a good practice going forward to optimize file size in the repo.